src/main/webapp/index.html

```diff
@@ -3,10 +3,13 @@
   <head>
     <meta charset="UTF-8">
     <title>Open Sesame</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>The Open Source Community is for <span id="emphasis">Everyone</span></h1>
+    <h1 class="display-1">The Open Source Community is for <span id="emphasis">Everyone</span></h1>
+    <spacer class="space"></spacer>
+    <button class="button">Mentorship Awaits</button>
   </body>
 </html>
```

Comment:
some tags don't do anything such as ```<html>, <head> and <body>``` the style guide talks a bit about removing these tags [here](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags). the documentation isn't super clear so it is an optional style requirement. I think code is a bit easier to read without the ```<html>``` and ```<head>``` tags. This is completely optional.

Reply:
This was the link Bill showed me for the window.onload functionality. https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload

Reply:
Thank you! I def feel comfortable removing the optional tags now :)